core/src/main/java/com/linecorp/armeria/internal/PathAndQuery.java

```diff
@@ -269,14 +263,15 @@ private static Bytes decodePercentsAndEncodeToUtf8(String value, int start, int
                     }
                 } else {
                     // If query:
-                    final Integer marker = RAW_CHAR_TO_MARKER.get(decoded);
-                    if (marker != null) {
+                    assert decoded < 256;
```

Comment:
I guess this isn't necessary because `AIIOBE` will be raised anyway if that ever happens.

Reply:
I see. Removed. :-)